modules/cloudtrail/remote-state.tf

```
@@ -8,3 +8,15 @@ module "cloudtrail_bucket" {
 
   context = module.this.context
 }
+
+module "account_map" {
+  source  = "cloudposse/stack-config/yaml//modules/remote-state"
+  version = "1.4.2"
+
+  component   = "account-map"
+  environment = var.account_map_environment_name
+  stage       = var.account_map_stage_name
+  privileged  = var.account_map_privileged
```

Get settings for `account-map` from `module.iam_roles`:

```suggestion
  environment = module.iam_roles.global_environment_name
  stage       = module.iam_roles.global_stage_name
```